kernel-open/common/inc/nv_uvm_interface.h

```diff
@@ -121,7 +121,7 @@ NV_STATUS nvUvmInterfaceSessionDestroy(uvmGpuSessionHandle session);
     nvUvmInterfaceDeviceCreate
 
     Creates a device object under the given session for the GPU with the given
-    UUID. Also creates a partition object for the device iff bCreateSmcPartition
```

Comment:
iff is probably not a typo: https://www.lexico.com/definition/iff

Reply:
Okay, fixing it

Reply:
Reverted it everywhere.